src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.tsx

```diff
@@ -229,6 +238,26 @@ const StatusForm: FunctionComponent<StatusFormProps> = ({
     }
   }, [escFunction, listenFor, unlisten])
 
+  useEffect(() => {
+    if (!emailTemplate) return
+
+    if (emailTemplate?.html) {
+      openEmailPreviewModal()
+      dispatch({ type: 'SET_EMAIL_TEMPLATE', payload: emailTemplate })
+    }
+  }, [emailTemplate, openEmailPreviewModal, dispatch])
+
+  useEffect(() => {
+    if (emailTemplateError) {
+      dispatch({
```

Comment:
This will dispatch an error to the local reducer instead of the global reducer. Whenever a `404` or `500` error status code is received, the `showGlobalNotification` action should be called. This way, the error is shown and it will allow the submit event to complete.